x-pack/plugin/monitoring/src/main/java/org/elasticsearch/xpack/monitoring/Monitoring.java

```diff
@@ -135,7 +135,7 @@ public Collection<Object> createComponents(
         IndicesService indicesService
     ) {
         final ClusterSettings clusterSettings = clusterService.getClusterSettings();
-        final CleanerService cleanerService = new CleanerService(settings, clusterSettings, threadPool, getLicenseState());
```

Comment:
`getLicenseState()` apparently has no side-effects, it doesn't throw or anything (and even if it did, we call it repeatedly below too)